packages/relay/src/lib/eth.ts

```diff
@@ -58,7 +58,7 @@ export class EthImpl implements Eth {
   static emptyArrayHex = '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347';
   static zeroAddressHex = '0x0000000000000000000000000000000000000000';
   static emptyBloom = "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
-  static defaultGas = EthImpl.numberTo0x(constants.TX_DEFAULT_GAS);
+  static defaultGas = EthImpl.numberTo0x(process.env.TX_DEFAULT_GAS ? Number.parseInt(process.env.TX_DEFAULT_GAS) : 400_000);
```

Comment:
Like this...

```suggestion
  static defaultGas = EthImpl.numberTo0x(process.env.TX_DEFAULT_GAS ? Number.parseInt(process.env.TX_DEFAULT_GAS) : constants.TX_DEFAULT_GAS);
```

Also, unrelated to this I realized today while debugging this class that since this properties are `static` somehow the `process.env` does not have our custom properties by time it loads, so it will always fallback to the default one. 😅 

One way to fix this could be to declare the property as `private readonly`  instead of `static`


```suggestion
  private readonly defaultGas = EthImpl.numberTo0x(process.env.TX_DEFAULT_GAS ? Number.parseInt(process.env.TX_DEFAULT_GAS) : constants.TX_DEFAULT_GAS);
```

<img width="1292" alt="image" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/828949a6-ac27-427c-9bb3-54a732339cad">

<img width="1271" alt="image" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/2dd672aa-4fcc-454d-a6e8-f76aa815c497">